docs/guides/update-guide/120-to-130.md

```diff
@@ -9,6 +9,16 @@ The following sections explain which adjustments must be made to migrate from Ca
 
 ## Server
 
+### Zeebe
+
+:::caution
+The docker image for Zeebe 1.3.0 does not set a system locale.
+This can lead to user data corruption, in particular with non-ascii characters (such as umlauts) in process variables.
```

Comment:
```suggestion
This can lead to problems with user data, like process variables, which use non-ASCII characters (such as umlauts).
```

My proposal is to not call this corruption, as this is makes users quite uncomfortable, wdyt?

Reply:
we agreed it is data corruption